.gitignore

```
@@ -100,4 +100,7 @@ lint/tmp/
 
 gas-report.txt
 
-contracts/test/fuzzing/crytic-export
\ No newline at end of file
+contracts/test/fuzzing/crytic-export
+
+# Generated on demand request data
+on-demand-request.json
```

```suggestion
on-demand-request.json
on-demand-request-source.js
```
Probably want to also ignore the source code